### PR TITLE
Created pop up after clicking on gift certificate icon

### PR DESCRIPTION
**Pull request by MickhaelSh**
## Description
Created pop up after clicking on gift certificate icon
#### Screenshots

giftcertificate

### Checklist

- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✅All tests passed locally
- [x] ✨My changes working with up-to-date admin and back-end part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

File: src/containers/modal-gift-certificate/tests/modal-gift-certificate.spec.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    act(() => {
+      fireEvent.click(dismissButton);
+    });
+
+    expect(setModalVisibility).toHaveBeenCalledWith(false);
+  });
+  it('should call func on dismiss', async () => {
+    const input = screen.getByRole('textbox');
+
+    act(() => {
```
Comment (Tolik170) [inline src/containers/modal-gift-certificate/tests/modal-gift-certificate.spec.js:58]: i think you don't need act to wrap fireEvent
https://testing-library.com/docs/example-input-event

File: src/pages/gift-certificate/operations/gift-certificate.mutations.js [hunk 2]
```diff
@@ -18,3 +18,18 @@ export const generateCertificate = gql`
     }
   }
 `;
+
+export const giftCertificateToEmail = gql`
+  mutation ($id: ID!, $email: String!, $oldEmail: String!, $language: Int!) {
```
Comment (Tolik170) [inline src/pages/gift-certificate/operations/gift-certificate.mutations.js:23]: maybe better create a type or use existing one

File: src/containers/modal-gift-certificate/modal-gift-certificate.js [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+        />
+        <div className={styles.buttonGroup}>
+          <Button
+            variant='contained'
+            disabled={gitftCertificateLoading}
+            type='submit'
+            data-testid='confirm'
+          >
+            {gitftCertificateLoading ? (
+              <Loader heightWrap='25px' height='20px' width='20px' />
```
Comment (Tolik170) [inline src/containers/modal-gift-certificate/modal-gift-certificate.js:87]: better move it to styles

File: src/components/confirm-dialog/confirm-dialog.styles.js [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+    borderBottom: '2px solid #E2E8F0',
+    fontSize: '20px',
+    color: theme.palette.black,
+    fontWeight: '700',
+    '@media (max-width: 500px)': {
+      padding: '5px'
+    }
+  },
+  closeIcon: {
+    color: '#808080',
```
Comment (Tolik170) [inline src/components/confirm-dialog/confirm-dialog.styles.js:52]: use closeModalIcon from palette

File: src/components/confirm-dialog/confirm-dialog.styles.js [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+        fontSize: '14px'
+      }
+    }
+  },
+  header: {
+    display: 'flex',
+    padding: '10px 0',
+    justifyContent: 'space-between',
+    boxSizing: 'border-box',
+    borderBottom: '2px solid #E2E8F0',
```
Comment (Tolik170) [inline src/components/confirm-dialog/confirm-dialog.styles.js:43]: use titleBorder from palette

File: src/components/confirm-dialog/confirm-dialog.js [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+  };
+
+  return (
+    <Modal
+      open={open}
+      onClose={() => handleClose(false)}
+      aria-labelledby='simple-modal-title'
+      aria-describedby='simple-modal-description'
+    >
+      <div className={styles.paper} data-cy='removing-modal'>
```
Comment (Tolik170) [inline src/components/confirm-dialog/confirm-dialog.js:30]: i don't see that you use this data-cy in tests.

**Review by Tolik170 — CHANGES_REQUESTED** [review]

**Review by dmytruk-y — APPROVED** [review]

**Review by Tolik170 — DISMISSED** [review]

**Review by dmytruk-y — APPROVED** [review]

**Review by Tolik170 — APPROVED** [review]

**Pull request merged**